src/log/MonologTarget.php

```diff
@@ -141,6 +153,33 @@ public function export(): void
         $logger->popProcessor();
     }
 
+    public function getDefaultHandlers(): array
+    {
+        $handlers = [];
+
+        if (App::isStreamLog()) {
+            $handlers[] = (new StreamHandler(
+                'php://stderr',
+                Level::Warning,
+                bubble: false,
+            ))->setFormatter($this->formatter);
+            $handlers[] = (new StreamHandler(
+                'php://stdout',
+                $this->level,
+                bubble: false,
+            ))->setFormatter($this->formatter);
+        } else {
+            $handlers[] = (new RotatingFileHandler(
+                App::parseEnv(sprintf('@storage/logs/%s.log', $this->name)),
+                $this->maxFiles,
+                $this->level,
+                filePermission: Craft::$app->getConfig()->getGeneral()->defaultFileMode,
+            ))->setFormatter($this->formatter);
+        }
+
+        return $handlers;
+    }
+
```

Comment:
`getDefaultHandlers` is not static because it needs access to `$this->formatter` and `$this->name`.

Its intended use is to be referenced in `handlers` when a Closure is used.